platforms/jvm/ear/src/main/java/org/gradle/plugins/ear/Ear.java

```diff
@@ -130,19 +137,24 @@ private OutputChangeListener outputChangeListener() {
         return getServices().get(OutputChangeListener.class);
     }
 
-    private void recordTopLevelModules(FileCopyDetails details) {
+    private SetProperty<EarModule> recordTopLevelModules(FileCopyDetails details) {
```

Comment:
:x: Why return a `SetProperty` here instead of a `Set`?

Reply:
Yes, we can use `Set`, changed to `Set`